packages/core/src/lib/message/version_0.ts

```diff
@@ -116,15 +120,19 @@ export class Encoder implements IEncoder {
  * messages.
  */
 export function createEncoder({
+  pubSubTopic = DefaultPubSubTopic,
```

Comment:
Note that `pubsubTopic` parameter is only for Status using static sharding so I'd put it last in the list of properties.